dev/io.openliberty.grpc.1.0.internal.shaded/bnd.bnd

```diff
@@ -0,0 +1,79 @@
+#*******************************************************************************
+# Copyright (c) 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+
+-sub: *.bnd
+
+grpcVersion=1.43.2
+
+bVersion=1.0
+
+Import-Package: \
+  !com.aayushatharva.brotli4j*,\
+  !com.github.luben.zstd,\
+  !com.google.protobuf.nano,\
+  !com.google.code.gson,\
+  !org.checkerframework,\
+  !com.ibm.websphere.monitor.jmx,\
+  !com.jcraft.jzlib,\
+  !com.ning.*,\
+  !com.oracle.svm.core.annotate,\
+  !io.netty.internal.tcnative,\
+  !lzma.*,\
+  !net.jpountz.*,\
+  !org.apache.log4j.*,\
+  !org.apache.logging.*,\
+  !org.bouncycastle.*,\
+  !org.conscrypt,\
+  !org.eclipse.jetty.*,\
+  !org.jboss.*,\
+  !reactor.blockhound.*,\
+  !sun.*,\
+  !com.squareup.okhttp,\
+  okio,\
+  !android.*,\
+  !kotlin,\
+  !kotlin.*,\
+  !org.openjsse.javax.net.ssl,\
+  !org.openjsse.net.ssl,\
+  *
+
+Export-Package: \
+  io.grpc.netty.*;version=${grpcVersion},\
+  io.grpc.okhttp.*;version=1.47.0,\
+  okio;version="1.0.0",\
+  okhttp3.*;version="1.0.0"
+  okhttp3;version="1.0.0"
+
+instrument.disabled: true
+
+-fixupmessages.missingexport: "Used bundle version * for exported package";is:=ignore
+
+-buildpath: \
+  io.grpc:grpc-netty-shaded;version=${grpcVersion},\
```

Comment:
Also, I'm not sure that we really want to bundle the shaded version of grpc-netty at all... we need to think about this.